fp/fp.py

```diff
@@ -25,30 +27,45 @@ def __init__(self, country_id=None, timeout=0.5, rand=False, anonym=False, elite
         self.google = google
         self.schema = 'https' if https else 'http'
 
-    def get_proxy_list(self):
+    def get_proxy_list(self, repeat):
```

Comment:
Just wanted to let you know that this should have been a keyword argument with a default value as in the case of `get`, instead of an argument. It breaks backward compatibility for a minor release.